docs/advanced-features/tabulator.md

```diff
@@ -195,41 +197,51 @@ else:
     print(f'Query did not succeed. Final state: {state}')
 ```
 
-## Unrestricted Access
+## Open Query
 
 > Available since Quilt Platform version 1.57
 
-By default, Tabulator is only accessible via a session provided by the Quilt Catalog,
-and the access is scoped to the permissions of the Catalog user associated with
-that session. However, an admin can enable **unrestricted access** to Tabulator,
-deferring all access control to AWS. The underlying data in S3 is accessed using
-the Tabulator's dedicated "unrestricted" role, which has read-only access to all
-the S3 buckets attached to the given stack. This allows querying the data directly
-from the AWS Console or Athena views, given the caller has the necessary permissions
-to access Athena resources associated with Tabulator.
+By default, Tabulator is only accessible via a session provided by the Quilt
+Catalog, and the access is scoped to the permissions of the Catalog user
+associated with that session. However, admins can choose to enable **open
+query** to Tabulator tables, deferring all access control to AWS, thus enabling
+access from external services.
+
+In this case, the underlying data in S3 is accessed using the Tabulator's
+dedicated "open query" role, which has read-only access to all the S3 buckets
+attached to that stack. This allows querying Tabulator from the AWS Console,
+Athena views or JDBC connectors -- as long as the caller has been granted the
+necessary permissions to access Athena resources associated with Tabulator.
 
 ![Tabulator Settings](../imgs/admin-tabulator-settings.png)
 
 ### Permissions & Configuration
 
-In order to access Tabulator in unrestricted mode, the caller must:
+In order to access Tabulator in open query mode, the caller must have
+permissions to:
 
-1. Provide a workgroup with output location set and compatible with that of the
-   Tabulator (`s3://${UserAthenaResultsBucket}/athena-results/non-managed-roles/`).
+- use a new or existing Athena workgroup
+- read and write to that workgroup's output location, e.g.
+  `s3://${UserAthenaResultsBucket}`
+- invoke the Tabulator Lambda function
+- read from the relevant Tabulator bucket (in order to access spill files, if
+  any)
 
-2. Have the following permissions:
+In addition, Tabulator itself must be able to write to the workgroup's output
+location. The easiest way to do this is by adding that bucket to the Quilt
+stack.
```

Comment:
this is not quite correct. adding a bucket to the stack won't solve anything. tabulator's "open access role" has hardcoded permissions for writing results to `${UserAthenaResultsBucket}/athena-results/*/*`, and therefore the provided workgroup **must** have compatible results location